Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM dwolla/jenkins-agent-core:alpine
+
+USER root
+ENV SONAR_RUNNER_HOME=/usr/lib/sonar-scanner
+ENV NODE_PATH=/usr/local/lib/node_modules
+ENV SONAR_SCANNER_VERSION=4.4.0.2170
+
+RUN apk add --no-cache curl grep sed unzip nodejs nodejs-npm bash && \
+    npm install -g typescript
+
+WORKDIR /usr/src
+
+RUN curl --insecure -o ./sonarscanner.zip -L https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-${SONAR_SCANNER_VERSION}-linux.zip && \
```

Comment:
Why is this marked with `--insecure`?

Reply:
Will remove this. Was following the SonarSource docker scanner and left this is. There should be no issues with certificate validation.